### ISSUE TITLE
Using StyleSheetServer within a Webpack Build

### ISSUE THREAD

**Issue by kohlmannj** [Khan/aphrodite#120]
Hi,

I realize this entire approach may be bad, so feel free to use that as an escape hatch.

I'm attempting to use `StyleSheetServer` to statically render a React component _within a Webpack entry point_. More specifically, I'm attempting to do this within a [html-webpack-plugin](https://github.com/ampedandwired/html-webpack-plugin) template, which gets compiled with Webpack as well.

For example:

**template.jsx:**
```jsx
import React from "react";
import ReactDOMServer from "react-dom/server";
import RootView from "./root-view.jsx";
import { StyleSheetServer } from "aphrodite";

module.exports = function (templateParams) {  
  const {html, css} = StyleSheetServer.renderStatic(() => (
    ReactDOMServer.renderToString(<RootView />, {})
  ));
  
  const templateStr = '<html><head>' +
    '<title>' + templateParams.htmlWebpackPlugin.options.title + '</title>' +
    '</head><body>' + '<style data-aphrodite>' + css.content + '</style>' +
    '<div id="app">' + html + '</div></body></html>';
  
  return templateStr;
};
```

**root-view.jsx:** [please see this Gist](https://gist.github.com/kohlmannj/7025ae7514b571eb43affb5d1c1de90c)

At any rate, when trying to render a React view in this way, html-webpack-plugin produces the following error when trying to build with Webpack 2.1.0-beta.19:

    ERROR in Template execution failed: Error: Cannot automatically buffer without a document

This is of course a telltale sign that I'm using `StyleSheetServer` incorrectly. I'm wondering if this is a side effect of attempting to use `StyleSheetServer` within a Webpack environment—is Webpack's static analysis and module bundling preventing `StyleSheetServer` from intercepting calls to the `css()` helper method, for example?

Thanks in advance for taking a look.

**Comment by xymostech** [comment]
Interesting. It looks to me like you're doing things correctly! I don't believe that it happening inside of a webpack bundle would change anything. Do you get a traceback of the error?

**Comment by kohlmannj** [comment]
@xymostech One moment, will prepare a limited test case repository and get back to you. Thanks so much.

**Comment by kohlmannj** [comment]
Good news! Looks like everything works, as it turns out. For posterity, my limited test case repository is here (including a README): https://github.com/nytpi/aphrodite-ltc

Seems I got bitten bad by rofrischmann/inline-style-prefixer#92, which I resolved by [installing additional devDependencies as suggested](https://github.com/rofrischmann/inline-style-prefixer/issues/92#issuecomment-233912228) in a reply to the issue.
